pkg/validation/validation.go

```diff
@@ -28,9 +28,17 @@ var CustomResourceDefinitionValidator = internal.CRDValidator
 var BundleValidator = internal.BundleValidator
 
 // OperatorHubValidator implements Validator to validate bundle objects
-// for OperatorHub.io requirements.
+// for OperatorHub.io requirements. This validator is deprecated.
 var OperatorHubValidator = internal.OperatorHubValidator
```

Comment:
If you format the deprecation message a bit different, I think IDEs and maybe even the GoDoc sites, pick up on it better:

```suggestion
// for OperatorHub.io requirements.
//
// Deprecated: Use OperatorHubV2Validator and StandardCategoriesValidator for equivalent functionality.
var OperatorHubValidator = internal.OperatorHubValidator
```

Reply:
nice catch